web/ui/static/js/graph.js

```diff
@@ -1,16 +1,18 @@
 var Prometheus = Prometheus || {};
-var graphs = [];
 var graphTemplate;
 
 var SECOND = 1000;
 
 Handlebars.registerHelper('pathPrefix', function() { return PATH_PREFIX; });
 
-Prometheus.Graph = function(element, options) {
+Prometheus.Graph = function(element, options, changeHandler, removeHandler) {
```

Comment:
(though it might make sense to decouple what the handler functions exactly do, so maybe the handle... names are better in the end)